app/components/header/view.html.erb

```diff
@@ -12,24 +12,23 @@
           </span>
         </span>
         <span class="govuk-header__product-name">
-          <%= service_name %>
+          <%= title %>
```

Comment:
Why the name change here? This is always going to be our service name isn't it? Probably not the title?